docs/application/native/guides/internationalization/i18n.md

```diff
@@ -872,6 +876,63 @@ using a Number format for the PluralFormat locale.
 If you also pass a locale object to constructor, this locale will affect formatted number in the returned message texts.
 
+<a name="date_interval"></a>
+## Usages of date intervals
+
+The DateInterval API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__MODULE.html) applications) allows you to [create date intervals](#date_interval_examples).
+
+DateInterval module represents time period between pair of `i18n_udate` values. This type contains
+number of milliseconds from 1 January 1970 00:00 UTC to desired date.
+
+### Format date intervals
+
+The DateIntervalFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__DATE__INTERVAL__FORMAT__MODULE.html) applications) allows you to [format created date intervals](#date_interval_examples).
+
+Purpose of this module is to enable formatting date intervals in a language-independent manner.Date Interval Format module formats a Date Interval into text as compactly as possible.
+For example, the date interval format from "Jan 11, 2008" to "Jan 18,. 2008" is
+"Jan 11-18, 2008" for English.
+
+There is no structural information in date time patterns. For any punctuations and
+string literals inside a date time pattern, we do not know whether it is just a separator,
+or a prefix, or a suffix. Without such information, so, it is difficult to generate
+a sub-pattern (or super-pattern) by algorithm. So, formatting a Date Interval is pattern-driven.
+
+A skeleton
+- only keeps the field pattern letter and ignores all other parts in a pattern, such as space,
+  punctuations, and string literals.
+- hides the order of fields.
+- might hide a field's pattern letter length.
+
+For those non-digit calendar fields, the pattern letter length is important, such
+as MMM, MMMM, and MMMMM; EEE and EEEE, and the field's pattern letter length is honored.
+
+The calendar fields we support for interval formatting are: year, month, date, day-of-week,
+am-pm, hour, hour-of-day, minute, and second (though we do not currently have specific
+interval format date for skeletons with seconds). Those calendar fields can be defined in
+the following order: year > month > date > hour (in day) > minute > second.
+
+The largest different calendar fields between 2 calendars is the first different calendar
+field in above order.
+
+For example: the largest different calendar fields between "Jan 10, 2007" and "Feb 20,
+2008" is year.
+
+For other calendar fields, the compact interval formatting is not supported. And the interval
+format will be fall back to fall-back patterns, which is mostly "{date0} - {date1}".
+
+There is a set of pre-defined static skeleton strings. There are pre-defined interval patterns
```

Comment:
1. I suggest to leave as it is, because it is more convenient to use ", when we have many usages in one place rather than italics.

2. Fixed

3. Fixed